test/invariant/fuzz/properties/PropertiesAllo.t.sol

```diff
@@ -203,131 +211,105 @@ contract PropertiesAllo is HandlersParent {
 
     ///@custom:property-id 13
     ///@custom:property pool manager can always change metadata
-    function prop_poolManagerCanAlwaysChangeMetadata(uint256 _idSeed, uint256 _seedManager, Metadata calldata _metadata)
-        public
-    {
+    function prop_poolManagerCanAlwaysChangeMetadata(uint256 _idSeed, Metadata calldata _metadata) public {
         _idSeed = bound(_idSeed, 0, ghost_poolIds.length - 1);
         uint256 _poolId = ghost_poolIds[_idSeed];
-        uint256 _managerIndex = _seedManager % (ghost_poolManagers[_poolId].length - 1);
-        address _manager = ghost_poolManagers[_poolId][_managerIndex];
 
-        (bool _success,) =
-            targetCallDefault(address(allo), 0, abi.encodeWithSelector(allo.updatePoolMetadata.selector, _poolId, _metadata));
+        (bool _success,) = targetCall(address(allo), 0, abi.encodeCall(allo.updatePoolMetadata, (_poolId, _metadata)));
+
+        bool _isManager;
+        for (uint256 _i; _i < ghost_poolManagers[_poolId].length; _i++) {
+            if (msg.sender == ghost_poolManagers[_poolId][_i]) {
+                _isManager = true;
+                break;
+            }
```

Comment:
shouldn't this be tidier as a function (ideally have it on the address type, so you could just use `if(msg.sender.isManager() ...)`?